model/src/main/kotlin/OrtResult.kt

```diff
@@ -405,7 +406,17 @@ data class OrtResult(
 
     private fun getPackages(): Set<CuratedPackage> = analyzer?.result?.packages ?: emptySet()
 
-    private fun getProjects(): Set<Project> = analyzer?.result?.projects ?: emptySet()
+    /**
+     * Return all [Project]s contained in this [OrtResult].
+     */
+    @JsonIgnore
+    fun getProjects(): Set<Project> = analyzer?.result?.projects ?: emptySet()
```

Comment:
Should we make this a `val` property? Prefer `.orEmpty()` instead of the `?:` fallback.

Reply:
done `.orEmpty()`. The property `projects` already exists but is of different type and should not be exposed.